src/main/cpp/CMakeLists.txt

```diff
@@ -26,13 +22,12 @@ set(CUDFJNI_BUILD_DIR
   "$ENV{CUDFJNI_BUILD_DIR}"
   CACHE STRING "path to libcudfjni build root"
 )
-set(CUDF_CPP_BUILD_DIR
-  "$ENV{CUDF_CPP_BUILD_DIR}"
-  CACHE STRING "path to libcudf build root"
-)
+
 # libcudf's kvikio dependency requires this to be set when statically linking CUDA runtime
 set(CUDA_STATIC_RUNTIME ON)
 
+set(CUDF_DIR "${CMAKE_SOURCE_DIR}/../../../thirdparty/cudf")
```

Comment:
Relative paths, and duplicate definitions of the paths by different means make it more error-prone. Why not propagate and rely on `$ENV{CUDF_PATH}` from Maven?

Reply:
Humnn, that is also possible if we need to build against external cudf repo. I'll revert that.

Reply:
Done. I've added a comment to clarify the purpose, so future refactor will not do this again.